plugin/hotswap-agent-spring-plugin/src/main/java/org/hotswap/agent/plugin/spring/transformers/ProxyReplacerTransformer.java

```diff
@@ -20,16 +20,18 @@
 
 import org.hotswap.agent.annotation.OnClassLoadEvent;
 import org.hotswap.agent.javassist.*;
+import org.springframework.beans.factory.support.RootBeanDefinition;
 
 /**
  * Transforms Spring classes so the beans go through this plugin. The returned beans are proxied and tracked. The bean
  * proxies can be reset and reloaded from Spring.
  *
  * @author Erki Ehtla
- *
  */
 public class ProxyReplacerTransformer {
-    public static final String FACTORY_METHOD_NAME = "getBean";
+    public static final String FACTORY_CREATE_METHOD_NAME = "createBean";
```

Comment:
Enhancing 'doCreateBean' is better.

Reply:
Please see `SpringPluginTest#hotswapBeanCreatedByIABPP` i just added ,this method complements the test scenario and proves that `doCreateBean` is not the right choice. 

You can change `ProxyReplacerTransformer#FACTORY_CREATE_METHOD_NAME` to `doCreateBean` and execute this test method. The key point of why failed is that the few lines of code above `doCreateBean`
```
try {
	// Give BeanPostProcessors a chance to return a proxy instead of the target bean instance.
	Object bean = resolveBeforeInstantiation(beanName, mbdToUse);
	if (bean != null) {
		return bean;
	}
}
```
 will directly return the `SpringCglibxxx` bean in some cases, instead of the `HOTSWAPAGENT$SpringCglibxxx` bean.

Reply:
I see. Thanks for pointing it out.